Taskfile.yaml

```diff
@@ -0,0 +1,93 @@
+version: 3
+
+# vars: # cannot be specified here due to https://github.com/go-task/task/issues/2108
+#   NESTED_MODULES: api
+#   API_DIRS: '{{.ROOT_DIR}}/api/v1beta1/...'
+#   MANIFEST_OUT: '{{.ROOT_DIR}}/api/crds/manifests'
+#   CODE_DIRS: '{{.ROOT_DIR}}/cmd/... {{.ROOT_DIR}}/internal/... {{.ROOT_DIR}}/test/... {{.ROOT_DIR}}/api/v1beta1/...'
+#   COMPONENTS: 'control-plane-operator'
+#   REPO_URL: 'https://github.com/openmcp-project/control-plane-operator'
+#   GENERATE_DOCS_INDEX: "true"
+
+includes:
+  shared:
+    taskfile: hack/common/Taskfile_controller.yaml
+    flatten: true
+    excludes:
+    - test
+    - test-envtest-dep
+    - build:bin:val:test
+    - build-raw
+    - build:bin:build-raw
+     # put task names in here which are overwritten in this file
+    vars:
+      NESTED_MODULES: api
+      API_DIRS: '{{.ROOT_DIR}}/api/v1beta1/...'
+      MANIFEST_OUT: '{{.ROOT_DIR}}/api/crds/manifests'
+      CODE_DIRS: '{{.ROOT_DIR}}/cmd/... {{.ROOT_DIR}}/internal/... {{.ROOT_DIR}}/test/... {{.ROOT_DIR}}/api/v1beta1/...'
+      COMPONENTS: 'control-plane-operator'
+      REPO_URL: 'https://github.com/openmcp-project/control-plane-operator'
+      GENERATE_DOCS_INDEX: "true"
```

Comment:
This repo doesn't even have a `docs` folder, so no need to generate an index for the files in it ;-)
```suggestion
```